### ISSUE TITLE
Click api with Proximity selector doesn't work in headless mode.

### ISSUE THREAD

**Issue by negiDharmendra** [getgauge/taiko#585]
### Expected behavior
`click` API work seamlessly in headless mode.

### Actual behavior
`click` API doesn't work in headless mode.

### Steps to reproduce
1. Create  HTML 

               <html>

                <body>
                    <span>Click with proximity</span>
                    <div>
                        <span onclick="displayText('Click works with text nodes.')">Click on text node</span>
                        <span>Click with proximity</span>
                    </div>
                    <span>Proximity marker</span><br />
                    <span onclick="displayText('Click works with proximity selector.')">Click with proximity</span>
                    <input onclick="displayText('Click works with text as value.')" value="Text as value" />
                    <input onclick="displayText('Click works with text as type.')" type="Text as type" />
                    <div onclick="displayText('Click works with text accross element.')">
                        Text <span>accross</span> elements
                    </div>
                    <script type="text/javascript">
                        function displayText(text) {
                            document.getElementById('root').innerText = text
                        }
                    </script>
                    <div style="height:1500px"></div>
                    <div id="root" style="background:red;"></div>
                    <span onclick="displayText('Click works with auto scroll.')">Show Message</span>
                </body>

                </html>

</html>

2. Run below script
   

          const { openBrowser, closeBrowser, goto, click,below,} = require('taiko');
         (async () => {
             try {
                   await openBrowser();
                   await goto(<path to html file>);
                   await click('Click with proximity', below('Proximity marker'));
           } catch (e) {
                 console.error(e);
           } finally {
                await closeBrowser();
          }
        })();

### Versions
```
Taiko: Version: 0.8.0 (Chromium: 76.0.3803.0) Commit :fe00a221ccb63c2667e162cef999cd741e299530

node : v10.15.3
```

**Note**
Seems to be an issue with chromium. As it renders text slightly different in headless mode. For more details check [puppeteer/issues/2410](https://github.com/GoogleChrome/puppeteer/issues/2410) and [chromium/issues/](https://bugs.chromium.org/p/chromium/issues/detail?id=744577)

**Comment by negiDharmendra** [comment]
Even after adding the [suggested](https://github.com/GoogleChrome/puppeteer/issues/2410#issuecomment-474738062) flag `--font-render-hinting` did not resolve this issue.

**Comment by zabil** [comment]
This is now fixed upstream and works on Taiko 1.0.6 with the script mentioned above.
